packages/ui-components/src/Tabs/TabHeader.js

```diff
@@ -250,6 +258,19 @@ export default class TabHeader extends Component{
     })
   }
 
+  tabSourceIsBegin = (index) => {
+    const isTabBeginDrag = index + 1
```

Comment:
it is better to use index + 1 as value  than create isTabBeginDrag variable.

for instance
```
this.setState({
      ++index 
})
```